src/components/UserCard/styles.ts

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+export const Card = styled.div`
+  background-color: ${({ theme }) => theme.detail};
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  max-width: 500px;
```

Comment:
```suggestion
  max-width: 768px;
```
Ajustar o max-with para o limite utilizado em outros media-queries, evitando este tipo de comportamento:
![image](https://user-images.githubusercontent.com/56613389/156234471-1b18fa28-7789-48bd-b402-721b3a57478d.png)